### DISCUSSION TITLE
有没有方法修改 cache ttl？

### DISCUSSION THREAD

**Discussion by ihotte** [IrineSistiana/mosdns#573] [General]
现有的 ttl 好像只能修改响应的 ttl，能否支持修改 cache 的 ttl。 

现在情况是如果上游返回一个超长的 ttl，会导致本地缓存长时间不过期，修改响应的 ttl 已经没啥实际意义了，除非重启 mosdns 才能清理缓存。

还有不知能否支持为特定域名（例如自用ddns域名）单独指定 ttl，默认的 ttl 设置对于经常变动的 ddns 域名来说不太理想

**Comment by urlesistiana** [comment]
缓存的ttl取自应答的ttl。你修改了应答的ttl就修改了缓存的ttl。

特定域名用 if (v4)，或 matches (v5) 匹配，然后执行不同的 ttl。
